package.json

```diff
@@ -31,6 +31,7 @@
     "libsodium": "^0.7.6",
     "libsodium-wrappers": "^0.7.6",
     "lodash": "^4.17.19",
+    "object-hash": "^2.2.0",
```

Comment:
Are we pulling in this dependency to avoid sorting, serializing, and hashing the payload with our own hashing util?

Reply:
yeah, this makes it easy and is well supported.

Reply:
ok

Reply:
That package is using a broken hashing algorithm by default. Fine for this use case since we are only using it as a local comparison of events. Just something to note.

Reply:
oh, I'm open to another option since this is still in review.

Reply:
This is fine. We are not relying on the hashing algorithms strength here. We just need a deterministic signature of the object.

Reply:
ok